dev_setup.sh

```diff
@@ -30,14 +30,13 @@ install_poetry() {
   if [ -d "${HOME}/Library/Application Support/pypoetry" ]; then
     echo "Poetry already installed. Skipping..."
   else
-    curl -sSL https://install.python-poetry.org | python
-    echo "Add 'export PATH="$HOME/.local/bin:$PATH"' to your shell config and restart
+    curl -sSL https://install.python-poetry.org | python3
```

Comment:
Hmm this is something janky with your config specifically. If you're in an active pyenv, `python` is shimmed to whatever version you're using. If you give it python3, it'll use the system python

Reply:
likely. I didn't mean to keep these changes